templates/default/less/variables.less

```diff
@@ -232,4 +232,23 @@
 @bs-datetimepicker-secondary-border-color: #ccc;
 @bs-datetimepicker-secondary-border-color-rgba: rgba(0, 0, 0, 0.2);
 @bs-datetimepicker-primary-border-color: white;
-@bs-datetimepicker-text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
\ No newline at end of file
+@bs-datetimepicker-text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
+
+//== Glyph
+@il-glyph-color: @link-color;
+@il-glyph-hover-color: @link-hover-color;
+@il-glyph-highlighted-color: @brand-warning;
+@il-glyph-highlighted-hover-color:  darken(@brand-warning, 15%);
+
+//== Counter
+@il-counter-padding: 2px 4px;
+@il-counter-font-size: 9px;
+@il-counter-margin-left: -4px;
+
+//== Status Counter
+@il-counter-status-bg: lighten(@mid-gray, 40%);
+@il-counter-status-margin-top: 10px;
+
+//== Novelty Counter
+@il-counter-novelty-bg: @brand-warning;
+@il-counter-novelty-margin-top: -5px;
```

Comment:
Thank you for your feedback, and interest in this PR!

Bootstrap variables and classes should be used as much as possible indeed! We did our best to do that with the counter. Nevertheless, their use in ILIAS asks for some slight adjustments. 

There is not official guideline to this, but from my point of UI, in cases like this, the best option is to write as little less as possible and introduce new classes acting as wrapper for bootstrap classes. This has the advantage, that skinning is a lot easier like this. From my point of view the meaning of il-counter-novelty-bg is easier to comprehense and manipulate compared to the corresponding line hidden in less. Further those variables will be directly manipulatable throught the GUI with ILIAS 5.2 enabling a better control over the GUI for administrators. My long term perspective is, that most aspects of skins, can be handled by manipulating variables with clear semantic naming.

The variable il-counter-novelty-margin-top is indeed an edge case. I thought about for quite some time how far we should go with using variables for such aspects. I am more than willing to discuss this. In my point of view, the key question should be, if this variable is likely to be changed in a custom skin.